README.md

```diff
@@ -17,76 +16,125 @@ _A table of contents to help people find things_
 - [Licence](#licence)
 
 ## Usage
-_Whatever usage instructions your component has. We've broken this down by Markup, JavaScript and Sass, but it depends how complex your component is._
+`o-message` uses Sass and Javascript to show and hide a message component.  
+It can be initialised declaratively if markup is provided on the page, or it can be initialised imperatively when using the [manual build process](http://origami.ft.com/docs/developer-guide/modules/building-modules/).
+
+By default, `o-message` initialises an alert message, which provides information in response to a user action. It currently has three themes, 'success', 'neutral' and 'error', and relies on the markup (or configuration) to determine certain aspects of each theme's styles.
 
 ### Markup
 
-_Common templating can go here, especially if there is only one template, but people can always check the demos for more._
+This is an example of the declarative way of instantiating an error message that spans **across a viewport**.
+
+```html
+<div class="o-message o-message--alert o-message--error" data-o-component="o-message">
+	<div class="o-message__container">
+		<div class="o-message__content">
+			<p class="o-message__content--highlight">Something went wrong!
```

Comment:
I don't think `o-message__content--highlight` or `o-message__content--detail` is BEMy. They're not modifiers of `o-message__content`, they're... sub elements? 

`o-message__highlight`, `o-message__detail`?